e2e/cypress/integration/messaging/message_edit_post_clear_text_spec.js

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @messaging
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+describe('MM-T2146 Remove all text from a post (no attachment)', () => {
```

Comment:
Unfortunately, TM4J ID at `describe` block will not work when saving test result. Please move to `it` block so it can be mapped correctly/automatically. Same with other instance.